lib/src/model.dart

```diff
@@ -4524,6 +4551,18 @@ class PackageGraph {
     return _inheritThrough;
   }
 
+  Set<Class> _invisibleAnnotations;
+
+  /// Returns the set of [Class] objects that are similar to pragma
+  /// in that we should never count them as documentable annotations.
+  Set<Class> get invisibleAnnotations {
+    if (_invisibleAnnotations == null) {
+      _invisibleAnnotations = new Set();
+      _invisibleAnnotations.add(specialClasses[SpecialClass.pragma]);
+    }
+    return _invisibleAnnotations;
+  }
```

Comment:
A matter of preferences, but this entire body can be written as:

  => _invisibleAnnotations ??= Set()..add(specialClasses[SpecialClass.pragma]);

(Code golf FTW :innocent:)